galrep.tex

```diff
@@ -425,12 +425,22 @@ \subsection{Other Groups}
 	Notice that the arguments above used many analytic facts about
 	geometry over $\C$ (e.g. homology, analytic structure) in order to
 	prove algebraic facts (e.g. the number of torsion points) about
-	$E(\overline{K})$. This is part of a more general concept denoted
+	$E(\overline{K})$. This is part of a more general concept called the
 	\emph{Lefschetz principle} which generally relates geometry over an
 	algebraically closed field of characteristic $0$ to geometry over
 	$\C$. For more on this see \cite[Ch.~VI.6]{silverman:aec}.
 \end{remark}
 
+\begin{remark}
+	The fact that $E[n] \approx (\Z/n\Z)^2$ holds over fields of
```

Comment:
"The fact that $E[n] \approx (\Z/n\Z)^2$ holds over fields of" -->
"In fact, we also have that $E[n] \approx (\Z/n\Z)^2$ over fields of"